cmd/kubebuilder/initproject/vendor.go

```diff
@@ -89,9 +89,20 @@ func depExists() bool {
 	return err == nil
 }
 
+// depManifestExists checks if DepManifestFile exists or not. It will panic
+// if it encounters unknown errors.
 func depManifestExists() bool {
 	_, err := os.Stat(depManifestFile)
-	return os.IsExist(err)
+	if err == nil {
```

Comment:
The caller `if Update && depManifestExists() {...}` only cares if the file exists not not, so, I am thinking 
```golang
     if  _, err := os.Stat(depManifestFile); err == nil {
             return true
    } 
    return false
```
will be enough. no need to take care of others (but subdivision If statements do can provide more info for debug).

Reply:
I am just being conservative here about not to overwrite the `gopkg.toml` incase of some edge case (like permission to read err or other I/O errors). Its better to warn user about the error and abort vendor update.

Reply:
Make sense.